Grbl_Esp32/src/WebUI/WebServer.cpp

```diff
@@ -1228,6 +1227,7 @@ namespace WebUI {
             sstatus        = "Upload failed";
             _upload_status = UploadStatusType::NONE;
         }
```

Comment:
This line could be omitted since the identical line is now present outside of the conditional

Reply:
Indeed I missed this one - fixed